splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/EventAppender.java

```diff
@@ -1,8 +1,18 @@
 package com.redhat.console.notifications.splunkintegration;
-public class EventAppender {
 
-    public static String append(String existing, String next) {
-        return existing + next;
-    }
+import org.apache.camel.AggregationStrategy;
+import org.apache.camel.Exchange;
+
+public class EventAppender implements AggregationStrategy {
 
+    public Exchange aggregate(Exchange oldExchange, Exchange newExchange) {
+        if (oldExchange == null) {
+            return newExchange;
+        }
+
+        String oldBody = oldExchange.getIn().getBody(String.class);
+        String newBody = newExchange.getIn().getBody(String.class);
+        oldExchange.getIn().setBody(oldBody + newBody);
+        return oldExchange;
```

Comment:
We should also think about what to do with headers during aggregation. This takes the headers of first message and then skips the rest.